VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -154,11 +154,11 @@ private void setCurrentUnit(CourseComponent component){
 
         environment.getDatabase().updateAccess(null, selectedUnit.getId(), true);
 
-        CourseComponent parent = component.getParent();
+        CourseComponent parentSectionComponent = component.getParent().getParent();
         String prefName = PrefManager.getPrefNameForLastAccessedBy(getProfile()
             .username, selectedUnit.getCourseId());
         final PrefManager prefManager = new PrefManager(MainApplication.instance(), prefName);
-        prefManager.putLastAccessedSubsection(parent.getId(), false);
+        prefManager.putLastAccessedSubsection(parentSectionComponent.getId(), false);
```

Comment:
I am removing this function and its call, as we are already taking care of this in the parent activity `CourseUnitNavigationActivity`.